2023.md

```diff
@@ -47,3 +47,4 @@
 * [Oxygen UI : A WSO2 Design System](https://medium.com/@savindiwaduge7/oxygen-ui-a-wso2-design-system-88eb7bfce857) by [Savindi Waduge](https://medium.com/@savindiwaduge7)
 * [Create A Secure Password Policy For Your Organization with Asgardeo](https://medium.com/@chanikaruchini-16/create-a-secure-password-policy-for-your-organization-with-asgardeo-6b7e813ef219) by [Chanika Ruchini](https://medium.com/@chanikaruchini-16)
 * [Introduction to OpenID Connect](https://medium.com/@asekatest1/introduction-to-openid-connect-a13acb6ac501) by [Aseka Warnakulasooriya](https://medium.com/@asekatest1)
+* [Events Are Opportunities!](https://medium.com/@Shaaali/events-are-opportunities-b8f17dde8907) by [Shalitha Pathiranage](https://medium.com/@Shaaali)
```

Comment:
Shall we also update the blog count for April?

Reply:
Done